R/tuneParams.R

```diff
@@ -204,6 +208,12 @@ tuneParams <- function(params,
                          data.position = "right",
                          data.intro = "Here you select the species whose parameters you want to change or whose properties you want to concentrate on."
                 ),
+                tags$br(),
```

Comment:
You don't need to include the resource selector in the intro. Also move the resource selector to `resourceControl`, see below.